src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ResourceService.scala

```diff
@@ -68,9 +67,20 @@ class ResourceService(
                 Resource(resourceTypeAdmin.name, ResourceId(rtName.value), Set.empty)
             }
           }
+          _ <- updateEffectivePolicyTables(samRequestContext, newOrUpdatedResourceTypeNames)
         } yield resourceTypes.values
     }
 
+  def updateEffectivePolicyTables(samRequestContext: SamRequestContext, newOrUpdatedResourceTypeNames: Set[ResourceTypeName]): IO[List[Unit]] = {
+    for {
+      response <- newOrUpdatedResourceTypeNames.filterNot(_ == SamResourceTypes.resourceTypeAdminName).toList.traverse { rtName => {
```

Comment:
We should include `resource-type-admin` here in case we ever add hierarchical roles to that resource type